config/deployer.sample.json

```diff
@@ -21,6 +21,9 @@
   "DBInstanceType": "db.r4.large",
   "DBUserName": "mmuser",
   "DBPassword": "mostest80098bigpass_",
+  "ExternalDBHost": "",
```

Comment:
Maybe we should us directly here something like ExternalDB only that get the DSN instead of forcing that, and if there is an external DSN defined use that instead of build that from the output config.